GrowthBook/src/commonMain/kotlin/com/sdk/growthbook/GrowthBookSDK.kt

```diff
@@ -120,16 +120,15 @@ class GBSDKBuilder(
     hostURL: String,
     attributes: Map<String, Any>,
     trackingCallback: GBTrackingCallback,
-    encryptionKey: String? = null
+    encryptionKey: String? = null,
+    private var networkDispatcher: NetworkDispatcher = CoreNetworkClient()
```

Comment:
Just to make sure: 
Will `Proguard` optimize and remove `CoreNetworkClient` from constructor if we pass our own `NetworkDispatcher`? otherwise we can't exclude `ktor` dependency

Reply:
Are you going to exclude ktor dependency from your own Android app or from Kotlin GrowthBook SDK?

Reply:
We are going to exclude `ktor` from our Android app

Reply:
Yes, `CoreNetworkClien`t is optimized and removed.
In order to minimize `CoreNetworkClient` you should NOT use rule 
`
-keep class com.sdk.growthbook.** { *; }
`

Reply:
Why you need `CoreNetworkClient` to be public instead of internal?

Reply:
In such implementation there is no need to make it public (I thought there would be no default param )

Reply:
Ok, I'll change `CoreNetworkClient ` to internal as it was before

Reply:
thanks for help

Reply:
my pleasure